public/javascripts/application.js

```diff
@@ -644,17 +644,3 @@ function thermometer() {
     }
   });
 }
-
-function updateCachedTokens() {
```

Comment:
Should this stay?

Reply:
Yes, I'm wondering if this was the result of a merge conflict.

Reply:
I think recent changes to the kudos function got overridden above (lines 457-468) as well, as did some recent CSS changes, which I'll comment on separately.